firmware/application/external/external.cmake

```diff
@@ -0,0 +1,16 @@
+# List of all the board related files.
+set(EXTCPPSRC
+
+	#pacman
+	external/pacman/main.cpp
```

Comment:
How does this work? Do you recombine the TUs later? I guess I expected to see one main per "app" however defined in CMake.

Reply:
There is no "per app". It is all compiled and linked in one go. The files in ${EXTCPPSRC} are added to all the other files in the main firmware.